1.0/en/0x10-C04-Infrastructure.md

```diff
@@ -28,12 +28,10 @@ Ensure cryptographic integrity and supply chain security through reproducible bu
 
 | # | Description | Level | Role |
 |:--------:|--------------------------------------------------------------------------------------------|:---:|:---:|
-| **4.2.1** | **Verify that** infrastructure-as-code is scanned with tools (tfsec, Checkov, or Terrascan) on every commit, blocking merges with CRITICAL or HIGH severity findings. | 1 | D/V |
-| **4.2.2** | **Verify that** container builds are reproducible with identical SHA256 hashes across builds and generate SLSA Level 3 provenance attestations signed with Sigstore. | 1 | D/V |
-| **4.2.3** | **Verify that** container images embed CycloneDX or SPDX SBOMs and are signed with Cosign before registry push, with unsigned images rejected at deployment. | 2 | D/V |
-| **4.2.4** | **Verify that** CI/CD pipelines use OIDC tokens from HashiCorp Vault, AWS IAM Roles, or Azure Managed Identity with lifetimes not exceeding organizational security policy limits. | 2 | D/V |
-| **4.2.5** | **Verify that** that Cosign signatures and SLSA provenance are validated during the deployment process before container execution and that verification errors cause the deployment to fail. | 2 | D/V |
-| **4.2.6** | **Verify that** build environments run in ephemeral containers or VMs with no persistent storage and network isolation from production VPCs. | 2 | D/V |
+
+| **4.2.1** | **Verify that** container builds are reproducible and produce signed provenance metadata that can be independently verified. | 1 | D/V |
```

Comment:
I suggest to replace in 4.2.1 - 4.2.3 the word `container` with something like `build artifacts (such as container etc)` to account for other resources than the containers as well.